shadow-dom/focus/focus-method-delegatesFocus.html

```diff
@@ -228,5 +228,13 @@
   assert_equals(document.activeElement, slottedToFirstSlot);
 }, "focus() on host with delegatesFocus & tabindex=0, #slottedToFirstSlot, #slottedToSecondSlot, #belowSlots  with tabindex=0");
 
+test(() => {
+  resetTabIndexAndFocus();
+  setTabIndex([aboveSlots, belowSlots], 0);
```

Comment:
Should host also get tabIndex 0?

Reply:
No, it shouldn't matter, the focus delegation will run (and stop at the first step) either way